recipes/flask-cache/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "flask-cache" %}
```

Comment:
@pmlandwehr added `flask-cache` and the feedstock lives in https://github.com/conda-forge/flask-cache-feedstock


I suggest you, @kwilcox, and @daf to become maintainers there.